content/en/guide/v10/components.md

```diff
@@ -94,7 +94,7 @@ In order to have the clock's time update every second, we need to know when `<Cl
 | `getSnapshotBeforeUpdate(prevProps, prevState)` | called just before `render()`. return value is passed to `componentDidUpdate`.
 | `componentDidUpdate(prevProps, prevState, snapshot)` | after `render()`
 
-> See [this diagram](https://twitter.com/dan_abramov/status/981712092611989509) to get a visual overview of how they relate to each other.
+> See [this diagram](https://imgur.com/bmfcRQm) to get a visual overview of how they relate to each other.
```

Comment:
I'd say get rid of the link entirely and embed the image on the page (make sure to download the image, don't hotlink)

Reply:
You can leave mention that it's from Dan's Twitter though